src/components/articles/reportArticleForm.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import * as PropTypes from 'prop-types';
```

Comment:
There is no need of `*` in this import. you can instead do `import PropTypes from 'prop-types;'`

Reply:
It has been worked on